sapi/cli/php_cli_server.c

```diff
@@ -754,12 +754,14 @@ static void sapi_cli_server_register_variables(zval *track_vars_array) /* {{{ */
 
 static void sapi_cli_server_log_write(int type, const char *msg) /* {{{ */
 {
+	FILE *out;
 	char buf[52];
 
 	if (php_cli_server_log_level < type) {
 		return;
 	}
 
+	out = type==PHP_CLI_SERVER_LOG_ERROR ? stderr : stdout;
```

Comment:
Nit

```suggestion
	out = type == PHP_CLI_SERVER_LOG_ERROR ? stderr : stdout;
```

Reply:
It seems to me that `PHP_CLI_SERVER_LOG_ERROR` is always defined as `2`, so the above conditional makes no sense. Maybe that should be an INI setting?